modules/standard/Types.chpl

```diff
@@ -17,27 +17,33 @@
  * limitations under the License.
  */
 
-// Types.chpl
-//
-// Standard type routines.
-//
+/*
+Functions related to predefined types.
 
-//
-// type predicates
-//
+This is an automatic module, so these functions are available
+to all Chapel programs.
+*/
+module Types {
+
+pragma "no doc" // joint documentation with the next one
 pragma "no instantiation limit"
 proc isType(type t) param return true;
+/* Indicates whether the argument is a type. */
```

Comment:
"Indicates" is still ambiguous.
I like to use the phrase: "Returns `true` if <predicate>; otherwise `false`.